spec.html

```diff
@@ -10350,7 +10350,7 @@
       <emu-note>
         <p>The following are the only restricted productions in the grammar:</p>
         <emu-grammar>
-          PostfixExpression[Yield] :
+          IncrementExpression[Yield] :
```

Comment:
I know I'm bikeshedding here, but calling this production `IncrementExpression` really bothers me when one of the two possibilities is a decrement. The SpiderMonkey AST called this an `UpdateExpression`. Could we go with that instead?